oss_src/unity/python/sframe/sys_util.py

```diff
@@ -65,6 +65,12 @@ def make_unity_server_env():
 
     ## set local to be c standard so that unity_server will run ##
     env['LC_ALL']='C'
+    # add certificate file
+    try:
+        import certifi
+        env['GRAPHLAB_FILEIO_ALTERNATIVE_SSL_CERT_FILE'] = certifi.where()
```

Comment:
Got it. I always tripped by the fact that you strip out the 'GRAPHLAB_' from environment variable name and use it inside unity_server as I always to a global search of original environment variable name!